### ISSUE TITLE
  Hi, I note that in the mnr-1 entry, t...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#3646]
<p>&nbsp;&nbsp;Hi,<br/>I note that in the mnr-1 entry, there is an incorrect annotation. dz175 is a nonsense allele that changes W208 to a stop codon. In addition, dz179 is a nonsense allele that changes N461 to a stop.<br/>Thank you,<br/>Hannes</p>

Reported by: Hann********* (hann*************************)  (obscured for privacy))
Submitted From: support <a href="http://www.wormbase.org//tools/support">/tools/support</a>

Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_2) AppleWebKit/600.3.18 (KHTML, like Gecko) Version/8.0.3 Safari/600.3.18

**Comment by tuli** [comment]
Dear Hannes,

Thank you for writing to WormBase.

I have checked the dz alleles in PMID  Salzberg Y et al. (2013) Cell
"Skin-derived cues control arborization of sensory dendrites in
Caenorhabditis ...."

The W at aa position 208 in mnr-1 is tgg. The paper does not specify
whether dz175  is an amber or opal mutation and so it was annotated
using ambiguity codes. The lesion is presented in WormBase as
W208Stop. Maybe I have misunderstood the issue you are reporting.

dz179 is, as you say, described as N461stop. The N at 461 is aat and
as I was not sure how this was mutated to a stop codon I did not add
flanking sequences but instead added a Remark stating "dz179 carries
an early stop codon". If you are able to send me the precise details
of this lesion I will add the curation.

Normally I would contact the authors when I am uncertain how to curate
alleles, but this one must have slipped through the net, so it's good
that you contacted us.

I look forward to hearing from you,

Mary Ann


~ Mary Ann Tuli

~ email: maryann.tuli@wormbase.org
~ Database Curator, WormBase

~ Biology Division
~ California Institute of Technology
~ 1200 E. California Blvd
~ Pasadena, CA 91125
